File: R/tests/testthat/test_prl_fictitious_rp.R
context("Test prl_fictitious_rp")
library(hBayesDM)

test_that("Test prl_fictitious_rp", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(prl_fictitious_rp(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})

File: Python/tests/test_prl_fictitious_rp.py
import pytest

from hbayesdm.models import prl_fictitious_rp


def test_prl_fictitious_rp():
    _ = prl_fictitious_rp(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()
